keps/sig-storage/1847-autoremove-statefulset-pvcs/kep.yaml

```diff
@@ -0,0 +1,43 @@
+title: Auto remove PVCs created by StatefulSet
+kep-number: 1847
+authors:  
+  - "@kk-src"
+  - "@dsu-igeek"
+  - "@mattcary"
+owning-sig: sig-storage
```

Comment:
I think this should be reversed. sig-apps should be owning the feature since this is enhancing the StatefulSet controller. sig-storage is in a participating role.